.github/actions/publish-nuget/action.yml

```diff
@@ -0,0 +1,126 @@
+name: Publish to NuGet
+description: Builds all .NET solutions in the given directory with static code analysis.
```

Comment:
:(

Reply:
?

Reply:
The description is a copy-paste leftover.